src/gauging.jl

```diff
@@ -10,13 +16,12 @@ function initialize_bond_tensors(ψ::ITensorNetwork; index_map=prime)
   return bond_tensors
 end
 
-"""Use an ITensorNetwork ψ, its bond tensors and gauging mts to put ψ into the vidal gauge, return the bond tensors and ψ_vidal."""
+"""Use an ITensorNetwork ψ, its bond tensors and belief propagation cache to put ψ into the vidal gauge, return the bond tensors and ψ_vidal."""
 function vidal_gauge(
   ψ::ITensorNetwork,
-  pψψ::PartitionedGraph,
-  mts,
-  bond_tensors::DataGraph;
-  eigen_message_tensor_cutoff=10 * eps(real(scalartype(ψ))),
+  bond_tensors::DataGraph,
+  bp_cache::BeliefPropagationCache;
```

Comment:
I think this should also get moved to a keyword argument, see my proposal here: https://github.com/mtfishman/ITensorNetworks.jl/pull/139#issuecomment-1954553578

Reply:
I think the devil is in the details there, and I find the current code logic to be confusing and hard to follow.